sound/soc/sof/ipc4-topology.c

```diff
@@ -1502,7 +1502,7 @@ static int sof_ipc4_prepare_mixer_module(struct snd_sof_widget *swidget,
 	int ret;
 
 	/* only 32bit is supported by mixer */
-	available_fmt->ref_audio_fmt = available_fmt->sink_audio_fmts;
+	available_fmt->format_list_to_search = available_fmt->sink_audio_fmts;
```

Comment:
nitpick: I would be hard pressed to see how the comment relates to the code. I know you have not changed the comment or how things work.